Jenkinsfile

```diff
@@ -532,7 +532,6 @@ stage('Non-default environments') {
 					String mavenBuildAdditionalArgs = ''' \
 							-pl !documentation \
 							-pl !integrationtest/mapper/orm-spring \
-							-pl !integrationtest/mapper/orm-jakarta-batch \
```

Comment:
🤞🏻 let's hope we won't get the timeout on our super faaaast DBs 🙈

Reply:
Good point. I just checked and we seem to be using a 30s timeout for most tests, so hopefully that will be enough...